3.13 Tweaks (#4)
MonarchDevelopment/MIPG-and-MSTR: MSTR.md

**Policy Additions**

**3.13A** In Head-to-Head tournaments, having the permission to look at an opponent’s card that the opponent can also look at is technically the same as that card being revealed. However in Multiplayer tournaments, that is not true. Being able to look at a card that an opponent can look at doesn’t give a player the right to reveal that card to everyone. According to Head-to-Head Tournament rules though, players are free to reveal cards that they can look at and this clashes with the concept of distinguishing between the Look and Reveal actions as defined in CR 701.16 (a through d). 

> One of the reasons why this happens in Head-to-Head is to avoid a player accidentally revealing hidden information and either them or the players that gained that information accidentally being penalized. In Multiplayer it is expected that when a player is dealing with hidden information that pertains to one of the opponent's that they are extra careful about it.

Author: ```suggestion
One of the reasons why this happens in Head-to-Head is to avoid a player accidentally revealing hidden information and either them or the players that gained that information accidentally being penalized. In Multiplayer it is expected that when a player is dealing with hidden information that pertains to one of their opponents, that they are extra careful about it.
```

One of the reasons want to allow players revealing hidden information they gained access to is to avoid a player accidentally revealing hidden information and either them or the players that gained that information accidentally being penalized. In Multiplayer it is expected that when a player is dealing with hidden information that pertains to one of their opponents, that they are extra careful about not physically revealing it. At the same time we also want to allow players to still be allowed to bluff about hidden information, so in any situation, players are allowed to verbally reveal any hidden information they may have gained.

> In Multiplayer tournaments, the following rules apply to physically revealing card faces in hidden zones when a player already knows their identity:

Author: ```suggestion
In Multiplayer tournaments, the following rules apply to physically revealing card faces in hidden zones when a player has permission to see them:
```

_Example: Alice resolves a Praetor’s Grasp targeting Bob. Alice may not reveal cards from Bob’s library while resolving the Praetor’s Grasp, neither may Bob. Alice may reveal the chosen card that she exiled face-down. In this example, Bob doesn't know the identity of the cards in their library, and Alice is not the owner of the cards. Also, Alice gained a permission to search Bob's library, not to reveal it, therefore she will be able to only physically reveal the face down exiled card after she chooses one._

_Example: Alice controls an Opposition Agent and is currently controlling Bob while Bob is searching their library. Although Alice can freely talk about cards in Bob's library with the remaining players, Alice cannot physically reveal the cards in Bob’s library to them. In this example, Alice is controlling Bob. However, controlling the player doesn’t grant Alice the right to make out-of-game choices or decisions. The choice to reveal hidden information at any time is granted by MTR 3.13, and as such is not an in-game choice or decision._

> _Example: Alice owns a Bane Alley Broker. she has activated it's ability exiling a Dark Ritual. At some point Bob gains control or Bane Alley Broker and also activates it exiling a Counterspell. Alice can still look at the Dark Ritual and she can reveal it. Bob can also look at Alice's Dark Ritual but he can't reveal it. If Charlie now gains control of Bane Alley Broker, both Alice, Bob and Charlie will be able to look at Dark Ritual, but still, only Alice will be able to reveal it, same as only Bob will be able to reveal Counterspell, in spite of the fact that both Bob and Charlie can look at it._

Author: ```suggestion
_Example: Alice owns a Bane Alley Broker. She activated its ability exiling a Dark Ritual. At some point Bob gains control or Bane Alley Broker and also activates it exiling a Counterspell. Alice can still look at the Dark Ritual and she can reveal it. Bob can also look at Alice's Dark Ritual but he can't reveal it. If Charlie now gains control of Bane Alley Broker, both Alice, Bob and Charlie will be able to look at Dark Ritual, but still, only Alice will be able to reveal it, same as only Bob will be able to reveal Counterspell, in spite of the fact that both Bob and Charlie can look at it._
```

**3.13A** In Head-to-Head tournaments, having the permission to look at an opponent’s card that the opponent can also look at is technically the same as that card being revealed. However in Multiplayer tournaments, that is not true. Being able to look at a card that an opponent can look at doesn’t give a player the right to reveal that card to everyone. According to Head-to-Head Tournament rules though, players are free to reveal cards that they can look at and this clashes with the concept of distinguishing between the Look and Reveal actions as defined in CR 701.16 (a through d).

> In Multiplayer tournaments, players may verbally reveal hidden information pertaining to other players that was shared with them. They may not do so physically without consent from the player that hidden information pertains to. Any hidden information pertains to the player that first gained it. In a situation where multiple players would gain hidden information simultaneously, consider the first to be the owner of the effect or action that caused it to be gained.

Author: ```suggestion
In Multiplayer tournaments, players may verbally reveal hidden information pertaining to other players that was shared with them. They may not do so physically without consent from the player that hidden information pertains to. Any hidden information pertains to the player that first gained it. In a situation where multiple players would gain hidden information simultaneously and one of them desires to reveal it, the Head Judge is the final arbiter regarding to which player that information pertains. When doing so, the Head Judge will abide by the philosophy of this rule such that no abuses happen.
```

_Example: Alice controls an Opposition Agent and is currently controlling Bob while Bob is searching their library. Although Alice can freely talk about cards in Bob's library with the remaining players, Alice cannot physically reveal the cards in Bob’s library to them. In this example, Alice is controlling Bob. However, controlling the player doesn’t grant Alice the right to make out-of-game choices or decisions. The choice to reveal hidden information at any time is granted by MTR 3.13, and as such is not an in-game choice or decision._

_Example: Alice owns a Bane Alley Broker. she has activated it's ability exiling a Dark Ritual. At some point Bob gains control or Bane Alley Broker and also activates it exiling a Counterspell. Alice can still look at the Dark Ritual and she can reveal it. Bob can also look at Alice's Dark Ritual but he can't reveal it. If Charlie now gains control of Bane Alley Broker, both Alice, Bob and Charlie will be able to look at Dark Ritual, but still, only Alice will be able to reveal it, same as only Bob will be able to reveal Counterspell, in spite of the fact that both Bob and Charlie can look at it._

> _Example: Bob is controlling Alice during her turn, due to resolving the activated ability of a Mindslaver. Alice draws an Ad Nauseam for the turn, then Alice casts and resolves Praetor's Grasp targetting Charlie, exiling a Lion's Eye Diamond. Bob has access to all of this hidden information, but this information still pertains to Alice, and as such, it's Alice's decision to reveal it or not, not Bob's. 

Author: ```suggestion
_Example: Bob is controlling Alice during her turn, due to resolving the activated ability of a Mindslaver. Alice draws an Ad Nauseam for the turn, then Alice casts and resolves Praetor's Grasp targetting Charlie, exiling a Lion's Eye Diamond. Bob has access to all of this hidden information, but this information still pertains to Alice, and as such, it's Alice's decision to reveal it or not, not Bob's._
```
